internal/pkg/flags/flags.go

```diff
@@ -133,6 +136,16 @@ func DefineFlagVar() *FlagVar {
 	flag.DurationVar(&flagVar.ManifestRequeueBusyInterval, "manifest-requeue-busy-interval",
 		DefaultManifestRequeueBusyInterval,
 		"determines the duration a Manifest in Processing state is enqueued for reconciliation.")
+	flag.Float64Var(&flagVar.ManifestRequeueJitterProbability, "manifest-requeue-jitter-probability",
+		DefaultManifestRequeueJitterProbability,
+		"determines the probability that jitter is applied to the requeue interval.")
+	flag.Float64Var(&flagVar.ManifestRequeueJitterPercentage, "manifest-requeue-jitter-percentage",
+		DefaultManifestRequeueJitterPercentage,
+		"determines the percentage range for the requeue jitter applied to the requeue interval. "+
+			"E.g., 0.1 means +/- 10% of the interval.")
+	flag.DurationVar(&flagVar.ManifestRequeueJitterDisableAfter, "manifest-requeue-jitter-disable-after-seconds",
```

Comment:
This flag is not strictly necessary. Either remove it or configure it's behavior so that the value of zero disables it i.e. the Jitter is always enabled.